scripts/download_headless.py

```diff
@@ -61,7 +61,8 @@ def download_headless(serial: str, output_path: str = None, dev: bool = False) -
 	r = requests.get(url, {'serial': serial, 'dev': str(dev).lower()})
 	results = json.loads(r.text)
 	if not results['ok']:
-		raise DownloadException('Download failed.')
+		message = results['message'] if 'message' in results else ''
+		raise DownloadException('Download failed: {}'.format(message))
```

Comment:
Good idea, but I think this would be a little better:
```python
		message = results.get('message', 'No information available')
		raise DownloadException('Download failed: {}'.format(message))
```

Reply:
Agreed, addressed in fdeaf3631212734ed4ce700b3be6c634e4d5d875.

Additionally I have no timeline for merging this so feel free to handle as y'all see fit. I made this change for debugging just now and figured it was a waste to create an issue so I made a quick pr with my changes.